cli/src/main/scala/com/codacy/analysis/cli/formatter/Formatter.scala

```diff
@@ -4,11 +4,12 @@ import java.io.{FileOutputStream, PrintStream}
 
 import better.files.File
 import com.codacy.analysis.core.model.Result
+import com.codacy.plugins.api.PatternDescription
 import org.log4s.{Logger, getLogger}
 
 trait FormatterCompanion {
   def name: String
-  def apply(stream: PrintStream): Formatter
+  def apply(outputStream: PrintStream, executionDirectory: File): Formatter
```

Comment:
Allow formatters to know where tool was run to be able to access fs.
Not sure if this is a great thing. :thinking: